publishing/media_overlay.html

```diff
@@ -18,6 +18,10 @@
                 <li data-feature="Web Animation API">Beyond text tracks, Web pages may contain many other time-based animations with which synchronization can be useful; the <a data-featureid="webanimations">Web Animations API</a> offers the tools needed to set up these synchronization points. This may become the basis for the Media Overlay work both in terms of model and terminology, as well as possible implementation work.</li>
 
                 <li data-feature="HTML Extensions">Media Overlay <em>may</em> require some simple extensions to HTML (e.g., additional attributes for Media Overlays). The <a data-featureid="html-extensions">HTML Extension Specifications</a> provides a good starting point for the roadmap to such extensions.</li>
+
+                <li data-feature="Timed Text Markup Language"><a data-featureid="ttml">TTML</a> Content may be used directly as a distribution format, for example, providing a standard content format to reference from a <code>track</code> element in an HTML document, or a <code>text</code> or <code>textstream</code> media element in a SMIL&nbsp;3.0 document.</li>
+
+                <li data-feature="WebVTT: The Web Video Text Tracks Format">The main use of <a data-featureid="webvtt">WebVTT</a> is for marking up external text track resources in connection with the HTML <code>track</code> element. WebVTT files provide captions or subtitles for video content, and also text video descriptions [MAUR], chapters for content navigation, and more generally any form of metadata that is time-aligned with audio or video content.</li>
```

Comment:
It would be good to make `[MAUR]` link to the [spec](https://www.w3.org/TR/media-accessibility-reqs/) itself.

Reply:
That's a good point, and I did not notice that. I merged the PR in the meantime, but maybe @iherman can prepare an update to test his now clean install? ;)